globus_action_provider_tools/exceptions.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Just dropping this comment at the top. This looks like a pretty nice method of unifying errors based on exceptions raised.